index_releases_sort_test.go

```diff
@@ -0,0 +1,95 @@
+package versionbundle
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestIndexReleasesSortByVersion(t *testing.T) {
+	testCases := []struct {
+		name          string
+		releases      []Release
```

Comment:
`s/Release/IndexRelease/g` :)

Reply:
Yep, just changed it in one place, should be done now PTAL.